docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I add payment information for my organization?
+
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Database usage pricing](https://www.featurebase.com/pricing){:target="_blank:}
+* [Available database sizes](https://docs.featurebase.com/docs/cloud/cloud-databases/cloud-db-shape/)
+
+## Important payment information
+
+{% include /cloud-org/cloud-payment-details.md %}
+
+## Add payment information for my organization
+
+* Click **Configuration** > **Upgrade to Paid**.
+* Choose your payment method: **Card** or **US bank account**
+* Enter your credit card information or your bank account details.
+* Click **Submit**.
+
+## Modify payment or cancel account
+
+* {% include contact-support.md %}
```

Comment:
The link for contact support points users to the contact form on the main website. Is this still correct given support is being handled via Discord?

Reply:
updated the include file